pkg/scheduler/framework/runtime/framework.go

```diff
@@ -938,7 +950,7 @@ func (f *frameworkImpl) RunScorePlugins(ctx context.Context, state *framework.Cy
 
 	// Run Score method for each node in parallel.
 	f.Parallelizer().Until(ctx, len(nodes), func(index int) {
```

Comment:
Sorry, one more thing.
Can we skip running `f.Parallelizer().Until` at L952 and L972 when all plugins are in Skip plugin?
Running as many goroutines as the Nods is kinda useless in that case.

Reply:
Yes we can skip this one, updated. And we can't skip third one cause we initialize `allNodePluginScores` in that one, we don't need to skip second one cause it uses `len(plugins)`

Reply:
> we don't need to skip second one cause it uses len(plugins)

👍